pkg/crds/wrangler/wrangler.go

```diff
@@ -0,0 +1,171 @@
+// Package wrangler is a TEMP package for common library code that should be ported to wrangler/pkg/crd
```

Comment:
I plan on having all the logic in this file moved to Wrangler so it can be shared with other repos.